connectors/v2/dreamfm.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+/**
+ * remove zero width characters & trim
+ * @param  {string} text to clean up
+ * @return {string} cleaned up text
+ */
+function cleanText(input) {
```

Comment:
I don't know if there is a real need for this cleaning. This site delivers tracks from Bandcamp, so it made sense to copy Bandcamp's connector behavior. Also, as vaporwave titles usually contain a lot of junk characters, a little sanitization is probably ok.